framework/db/ActiveQuery.php

```diff
@@ -122,6 +122,16 @@ public function init()
         $this->trigger(self::EVENT_INIT);
     }
 
+    /**
+     * returns connection object from `modelClass` if `$db` is null otherwise returns `$db`
+     * @param null|Connection your custom connection object
+     * @return Connection returns a connection object base on input
+    */
+    protected function finalConObj($db){
```

Comment:
```suggestion
    protected function ensureConnection($db)
    {
```

Reply:
`finalConObj` is `final connection object`. like the sentence `final decision`